src/transformers/generation/tf_logits_process.py

```diff
@@ -569,7 +569,7 @@ def _force_token(generation_idx):
             batch_size = scores.shape[0]
             current_token = self.force_token_array[generation_idx]
 
-            new_scores = tf.ones_like(scores, dtype=scores.dtype) * -float("inf")
```

Comment:
Is it possible to take the min value for a given dtype, e.g. as we do in torch with:
```py
min_dtype = torch.finfo(dtype).min
```

Reply:
+1